package-lock.json

```diff
@@ -9167,7 +9167,7 @@
     },
```

Comment:
I think this inclusion of package-lock.json changes was probably accidental? It seems unrelated to the topic of the PR (this commit was added after I did my review, sorry).